test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set (CMAKE_CXX_STANDARD 11)
+
+add_executable(
+        unit_tests
+        basic_tests.cpp
+)
+
+target_link_libraries(
+        unit_tests
+        duckx
+)
+
+add_test(
+        NAME
+        unit
+        COMMAND
+        ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}/test/unit_tests
```

Comment:
This should just be the target name. Also the indent width is inconsistent with the root CMakeLists.txt

Reply:
Yeah realised that when build was failing, will fix indentation aswell.